Sources/MessagesListViewController.swift

```diff
@@ -9,12 +9,15 @@
 import UIKit
 
 class MessagesListViewController: UIViewController, UITextFieldDelegate, UITableViewDelegate, UITableViewDataSource {
+    
+    @IBOutlet var textField: UITextField!
+    @IBOutlet var tableView: UITableView!
 	
 	override func viewDidLoad() {
 		super.viewDidLoad()
 		let nc = NSNotificationCenter.defaultCenter()
 		nc.addObserver(self,
-		               selector: #selector(messagesWereUpdated(_:)),
+		               selector: #selector(MessagesListViewController.messagesWereUpdated(_:)),
```

Comment:
AFAIK, this is not necessary, and I prefer to omit extra type info when it's not necessary.